src/System.Management.Automation/engine/remoting/common/RemoteSessionNamedPipe.cs

```diff
@@ -858,7 +869,8 @@ internal static void RunServerMode(string configurationName)
         /// Creates the process named pipe server object singleton and
         /// starts the client listening thread.
         /// </summary>
-        internal static void CreateIPCNamedPipeServerSingleton()
+        /// <param name="debugPipeName">Name of the pipe name to use.</param>
```

Comment:
`debugPipeName` name presumes IPC is for debugging, but that is not necessarily the case.  I think we should use a different name like `customPipeName` or `IPCPipeName`.